app/modules/asset_groups/models.py

```diff
@@ -833,8 +833,16 @@ def remove_annotation(self, annot_guid):
             ):
                 self.config['encounters'][encounter_num]['annotations'].remove(annot_guid)
 
+    # will default to hotspotter if no idConfigs provided by AGS
     def get_id_configs(self):
-        return self.config.get('idConfigs', [])
+        return self.config.get(
+            'idConfigs',
+            [
+                {
+                    'algorithms': ['hotspotter_nosv'],
```

Comment:
This is a nice idea but breaks the rule that there should be no platform specific content in the code and this is definitely zebra specific. 
Instead use ia_config_reader.get(f'_identifiers') to see what is in the IA config and use the first one. 
Also this functionality is also more correct in the sightings model than in the AGS code as that is the place where the algorithms are used.

Reply:
As I was doing a change in this area too, I put the fix in the sighting code that uses the first identifier in the IA_json:
 https://github.com/WildMeOrg/houston/pull/564/files#diff-e80c56afa47650611e07af0b11959dfa836cad5f07c1bc1d8ffb2c122ee3c2e3R1055-R1062

Reply:
updated; thanks for the template for this.